primitives/src/runtime_api.rs

```diff
@@ -221,5 +220,9 @@ sp_api::decl_runtime_apis! {
 		/// Returns all onchain disputes.
 		#[api_version(3)]
 		fn disputes() -> Vec<(v2::SessionIndex, v2::CandidateHash, v2::DisputeState<v2::BlockNumber>)>;
+
+		/// Returns execution environment parameter set for the session.
+		#[api_version(3)]
```

Comment:
We might have missed the train for version 3. The safest way is to add it to version 4.